ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_search.py

```diff
@@ -295,3 +295,39 @@ def test_paddle_topk(
         sorted=sorted,
         test_values=False,
     )
+
+
+# where
+@handle_frontend_test(
+    fn_tree="paddle.where",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_dim_size=1,
+    ),
+)
+def test_paddle_where(
+    *,
+    dtype_and_x,
+    x1,
+    x2,
+    name,
```

Comment:
Tests are failing due to this. For `x1` and `x2`, if you're going to populate them from `dtype_and_x` anyway, they don't need to be inputs to the function. For `name`, this is an unused argument to the frontend function, so you don't need to include it in testing at all